contrib/redwood/plugins.yml

```diff
@@ -32,3 +40,28 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: mfe-extensions
+  src: tutor-contrib-mfe-extensions>=18.0.0,<19.0.0
+  url: https://github.com/eduNEXT/tutor-contrib-mfe-extensions
+  author: eduNEXT <technical@edunext.co>
+  maintainer: eduNEXT <technical@edunext.co>
+  description: |
+    Tutor plugin to extend the MFE plugin.
+
+- name: pod-autoscaling
+  src: tutor-contrib-pod-autoscaling>=18.0.0,<19.0.0
+  url: https://github.com/eduNEXT/tutor-contrib-pod-autoscaling
+  author: eduNEXT <technical@edunext.co>
+  maintainer: eduNEXT <technical@edunext.co>
+  description: |
+    Tutor plugin enable pod-autoscaling mechanisms for OpenedX installations in Kubernetes.
```

Comment:
Can I suggest not replicating the bad pattern (that I created) to start descriptions with "Tutor plugin"? In most cases you can just remove that part of the description :)